cli/gh-pr-filters.go

```diff
@@ -207,7 +208,7 @@ func GetFilterForLabels(labels []string, and bool) *Filter {
 			for filterLabel, negate := range filterLabelMap {
 				_, found := labelMap[filterLabel]
 
-				// nolint:gocritic
+				//nolint:gocritic
```

Comment:
cli/gh-pr-filters.go:210:5: directive `// nolint:gocritic` should be written without leading space as `//nolint:gocritic` (nolintlint)
```
                                // nolint:gocritic
                                ^
```